src/controllers/signup.js

```diff
@@ -7,12 +7,18 @@ exports.get = (req, res) => {
 
 exports.post = (req, res) => {
   const { username, password } = req.body;
-  newUser.create({ username, password })
-    .then(console.log('Success!'))
-    .catch((error) => {
-      console.log(error);
+  bcrypt
+    .hash(password, 10)
+    .then(password => newUser.create({ username, password }))
+    .then(() => res.redirect('/thankYou'))
```

Comment:
might be nice to render `thankYou` with the user's username? just an idea no worries if not

`<h1>Thanks for signing up {{username}}!</h1> `